openedxscorm/scormxblock.py

```diff
@@ -24,23 +29,30 @@ def _(text):
 
 logger = logging.getLogger(__name__)
 
+# importing directly from settings.XBLOCK_SETTINGS doesn't work here... doesn't have vals from ENV TOKENS yet
+scorm_settings = settings.ENV_TOKENS['XBLOCK_SETTINGS']['ScormXBlock']
+SCORM_FILE_STORAGE_TYPE = scorm_settings.get("SCORM_FILE_STORAGE_TYPE", "django.core.files.storage.default_storage")
+SCORM_MEDIA_BASE_URL = scorm_settings.get("SCORM_MEDIA_BASE_URL", "/scorm")
+mod, store_class = SCORM_FILE_STORAGE_TYPE.rsplit('.', 1)
+scorm_storage_module = importlib.import_module(mod)
+scorm_storage_class = getattr(scorm_storage_module, store_class)
+if SCORM_FILE_STORAGE_TYPE.endswith('default_storage'):
+    scorm_storage_instance = scorm_storage_class
+else:
+    scorm_storage_instance = scorm_storage_class()
```

Comment:
What if we check that `scorm_storage_class` is callable or not?

Reply:
I think this thing is already being handled in the check.

Reply:
Yes it is. I meant what if we checked for it being callable instead of checking the end of type string? up to  you.

Reply:
It seems more easy and readable to me this way.